### PR TITLE
save result with trigger id

### PR DESCRIPTION
**Pull request by pnuckowski**
- [x] save result per trigger
- [x] get debounce from trigger
- [x] scheduled and execution time in cassandra
- [x] save `alert_sent` to cassandra

### CODE REVIEW

**Comment by ch40s3k** [comment]
:+1:

**Comment by pawellesniewski** [comment]
:ok_hand:

File: alamo_alerter/backend/cassandra.py [hunk 1]
```diff
@@ -48,19 +50,26 @@ def get_result(self, check_id, count):
...
+                    insertion_time: object):
         """Save check result.
 
         :param check_id: check id
-        :param insertion_time: time when check was analyzed
-        :param bool is_failed: check result as boolean value
-        :param str message: check result message
+        :param dict trigger: trigger definition
+        :param scheduled_time: time when check was scheduled
+        :param execution_time: time when worker finish execution
```
Comment (deejay1) [inline alamo_alerter/backend/cassandra.py]: worker finished execution

File: alamo_alerter/alerter.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 from datetime import datetime
 from collections import deque
 
+from pytz import utc as pytz_utc
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: pytz is missing in requirements, also don't dateutil and pytz do essentially the same stuff?

**Comment by deejay1** [comment]
:-1:

**Pull request merged**